cpp/zotero_harvester.cc

```diff
@@ -152,7 +155,14 @@ void ParseCommandLineArgs(int * const argc, char *** const argv, CommandLineArgs
             return; // intentional early return
         }
         case CommandLineArgs::SelectionMode::JOURNAL:
-            commandline_args->selected_journals_.emplace(current_arg);
+            if (StringUtil::StartsWith(current_arg, "--config-overrides=")) {
+                const std::string config_overrides(current_arg + __builtin_strlen("--config-overrides="));
+                FileUtil::AutoTempFile tempfile("/tmp/ATF", "", false);
```

Comment:
Kind iffy about using a temp file here. The "right" way to go about it would be to modify the `IniFile` class to support the direct passing of file contents to its constructor, but it's more a "good-to-have" thing at this point.

Reply:
I think this requires lots of changes and should be part of a different pull request.